tests/Integration/Behaviour/Features/Scenario/Product/bulk_duplicate_product.feature

```diff
@@ -33,69 +33,67 @@ Feature: Duplicate product from Back Office (BO).
       | meta description[en-US] |                    |
       | meta keywords[en-US]    | sup,1              |
       | shops                   | [shop1]            |
-    Given I add product "product1" with following information:
+    And I add product "product1" with following information:
       | name[en-US] | smart sunglasses   |
       | name[fr-FR] | lunettes de soleil |
       | type        | standard           |
     And I add product "product2" with following information:
       | name[en-US] | Reading glasses |
       | name[fr-FR] | lunettes        |
       | type        | standard        |
-    And I update product "product1" basic information with following values:
-      | description[en-US]       | nice sunglasses            |
-      | description[fr-FR]       | belles lunettes            |
-      | description_short[en-US] | Simple & nice sunglasses   |
-      | description_short[fr-FR] | lunettes simples et belles |
+    And I update product "product1" with following values:
```

Comment:
Haha, this is huge!!! But it feels much better than all the previous split steps, it makes more sense that we can do this in a single one